docs/slack.rst

```diff
@@ -275,13 +275,47 @@ In addition to the guidance that applies to all participants, we expect self-pro
   If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
   Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.
 
+Common Self-Promotion Situations
+++++++++++++++++++++++++++++++++
+
+Here are some examples of common self-promotion situations and how to self-promote responsibly (and irresponsibly):
+
+- You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion.
+
+  **Do** share it in **community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.
+
+  **Don't** cross-post that content across multiple channels or use the **community-showcase** (or any other) channel for link-farming or drive-by engagement only.
+
+- Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event).
+
+  **Do** share it in #community-events and tell others your connection to the event.
+
+  **Don't** obscure the purpose of the event, your relationship to the employer, or the costs associated to attendance.
+
+- Someone else has asked a question and you're the author of an article, blog, book, or similar resource that answers the question.
+
+  **Do** reply by briefly summarizing your work, noting your authorship, and linking to the resource.
+
+  **Don't** cross-post a reply outside the thread or channel or send the person asking the question a direct message.
+
+- A moderator has raised one or more of your posts as violating the Self-Promotion Guidelines, and provided you with feedback and guidance.
```

Comment:
style nit

Since we use sentence case, there are no capital-G guidelines

```suggestion
- A moderator has raised one or more of your posts as violating the self-promotion guidelines, and provided you with feedback and guidance.
```